### ISSUE TITLE
any interest in merging?

### ISSUE THREAD

**Issue by ttdonovan** [maiha/try.cr#3]
I see you've done the work for a Try monad. Is there any interest in merging your efforts into this lib?

https://github.com/dry-cr/dry-monads

I've gotten busy and never when back to complete the Try. If there is going to a monad lib for Crystal it would be beneficial we all started to collaborate.

**Comment by maiha** [comment]
@ttdonovan 
Yep, I researched monadic libs for Crystal last year, and found 4 or 5 projects.
They should be merged into one for Crystal community!

Although I've nowadays gotten busy too, I'd like to try merge them within 2017 or before `crystal-1.0`.
Thanks for contacting me. Let me discuss it again!
